Add sprint-integrations to no-error test of werker

command `./gradlew clean check --parallel -x test -PcheckstyleVersion=${CS_POM_VERSION}`

diff - https://github.com/romani/checkstyle/commit/159c5dd123299ea1f737e94d5bd09ab5cbc7eb44

full log:
```
./.ci/wercker.sh no-error-spring-integration
CS_version: 8.28-SNAPSHOT
Cloning into 'spring-integration'...
remote: Enumerating objects: 594, done.
remote: Counting objects: 100% (594/594), done.
remote: Compressing objects: 100% (295/295), done.
remote: Total 274708 (delta 105), reused 514 (delta 80), pack-reused 274114
Receiving objects: 100% (274708/274708), 53.53 MiB | 2.12 MiB/s, done.
Resolving deltas: 100% (127834/127834), done.
Checking connectivity... done.
Downloading https://services.gradle.org/distributions/gradle-6.0-bin.zip
.........10%.........20%.........30%.........40%.........50%.........60%.........70%.........80%.........90%.........100%

Welcome to Gradle 6.0!

Here are the highlights of this release:
 - Substantial improvements in dependency management, including
   - Publishing Gradle Module Metadata in addition to pom.xml
   - Advanced control of transitive versions
   - Support for optional features and dependencies
   - Rules to tweak published metadata
 - Support for Java 13
 - Faster incremental Java and Groovy compilation
 - New Zinc compiler for Scala
 - VS2019 support
 - Support for Gradle Enterprise plugin 3.0

For more details see https://docs.gradle.org/6.0/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
> Task :spring-integration-test-support:checkstyleMain FAILED
> Task :spring-integration-file:checkstyleMain FAILED
> Task :spring-integration-feed:checkstyleMain FAILED
> Task :spring-integration-core:checkstyleMain FAILED
> Task :spring-integration-event:checkstyleMain FAILED

FAILURE: Build completed with 5 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spring-integration-test-support:checkstyleMain'.
> Could not resolve all files for configuration ':spring-integration-test-support:checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:8.28-SNAPSHOT.
     Searched in the following locations:
       - https://repo.spring.io/libs-milestone/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://repo.spring.io/libs-milestone/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
       - https://repo.spring.io/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://repo.spring.io/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
     Required by:
         project :spring-integration-test-support

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spring-integration-file:checkstyleMain'.
> Could not resolve all files for configuration ':spring-integration-file:checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:8.28-SNAPSHOT.
     Searched in the following locations:
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
     Required by:
         project :spring-integration-file

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spring-integration-feed:checkstyleMain'.
> Could not resolve all files for configuration ':spring-integration-feed:checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:8.28-SNAPSHOT.
     Searched in the following locations:
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
     Required by:
         project :spring-integration-feed

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spring-integration-core:checkstyleMain'.
> Could not resolve all files for configuration ':spring-integration-core:checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:8.28-SNAPSHOT.
     Searched in the following locations:
       - https://repo.spring.io/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://repo.spring.io/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
     Required by:
         project :spring-integration-core

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

5: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spring-integration-event:checkstyleMain'.
> Could not resolve all files for configuration ':spring-integration-event:checkstyle'.
   > Could not find com.puppycrawl.tools:checkstyle:8.28-SNAPSHOT.
     Searched in the following locations:
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/maven-metadata.xml
       - https://oss.jfrog.org/artifactory/libs-snapshot/com/puppycrawl/tools/checkstyle/8.28-SNAPSHOT/checkstyle-8.28-SNAPSHOT.pom
     Required by:
         project :spring-integration-event

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.0/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 2m 30s
167 actionable tasks: 131 executed, 36 up-to-date

```

--- a/.ci/wercker.sh
+++ b/.ci/wercker.sh
@@ -152,6 +152,20 @@ no-error-strata)
   rm -rf Strata
   ;;
 
+no-error-spring-integration)
+  set -e
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  checkout_from https://github.com/spring-projects/spring-integration.git
+  cd .ci-temp/spring-integration
+  PROP_MAVEN_LOCAL="mavenLocal"
+  PROP_CS_VERSION="checkstyleVersion"
+  ./gradlew clean check --parallel -x test -P$PROP_MAVEN_LOCAL -P$PROP_CS_VERSION=${CS_POM_VERSION}
+  cd ../
+  rm -rf spring-integration
+  ;;
+
 no-exception-struts)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)

--- a/wercker.yml
+++ b/wercker.yml
@@ -8,6 +8,10 @@ build:
         export MAVEN_OPTS="-Dmaven.repo.local=${WERCKER_CACHE_DIR}"
         mvn -version
         echo "------"
+        echo "Link ${WERCKER_CACHE_DIR} to /root/.m2/repository for Gradle builds"
+        mkdir -p /root/.m2
+        ln -s ${WERCKER_CACHE_DIR} /root/.m2/repository
+        echo "------"
         du -hs ${WERCKER_CACHE_DIR}
         echo "------"
         du -hs ${WERCKER_CACHE_DIR}/* | sort -h
@@ -101,6 +105,16 @@ build:
         if [[ $RUN_JOB == 1 ]]; then
           echo "Command: ./.ci/wercker.sh no-error-equalsverifier"
           ./.ci/wercker.sh no-error-equalsverifier
+        else
+          echo "build is skipped ..."
+        fi
+
+  - script:
+      name: NoErrorTest - Spring Integration
+      code: |
+        if [[ $RUN_JOB == 1 ]]; then
+          echo "Command: ./.ci/wercker.sh no-error-spring-integration"
+          ./.ci/wercker.sh no-error-spring-integration
         else
           echo "build is skipped ..."
         fi

